choreolib/src/main/java/choreo/auto/AutoFactory.java

```diff
@@ -194,7 +194,6 @@ AutoTrajectory trajectory(String trajectoryName, AutoRoutine routine) {
     if (optTrajectory.isPresent()) {
       trajectory = optTrajectory.get();
     } else {
-      DriverStation.reportError("Could not load trajectory: " + trajectoryName, false);
```

Comment:
Should be replaced with alert?

Reply:
I removed those report error's because im pretty sure loading the optional trajectory through Choreo.loadTrajectory will already display an alert, and didn't want to duplicate this.